src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php

```diff
@@ -105,7 +104,7 @@ private function getClassCandidates(string $class): array
                 }
             }
 
-            if ($function[0] instanceof ComposerClassLoader || $function[0] instanceof SymfonyClassLoader) {
+            if ($function[0] instanceof ComposerClassLoader) {
```

Comment:
this should be kept: the class exists in cross components situations

Reply:
Can you give an example?

Reply:
any project that uses debug v4 + class-loader v3.4